docs/maui/behaviors/progressbar-animation-behavior.md

```diff
@@ -85,9 +85,9 @@ class ProgressBarAnimationBehaviorPage : ContentPage
 
 |Property  |Type  |Description  |
 |---------|---------|---------|
-| Progress | Double  | New Progress value to animate to. |
+| Progress | Double  | New Progress value to animate to as a percentage with 1 being 100% so 0.75 is 75% |
 | Length | uint | Duration in miliseconds |
-| Easing | enum | `enum` that controls the `Easing`, allows you to specify a transfer function that controls how animations speed up or slow down. You can find more details on [Easing here](https://docs.microsoft.com/xamarin/xamarin-forms/user-interface/animation/easing) |
+| Easing | enum | `enum` that controls the `Easing`, allows you to specify a transfer function that controls how animations speed up or slow down. You can find more details on [Easing here](/xamarin/xamarin-forms/user-interface/animation/easing) |
```

Comment:
This should point to the .NET MAUI docs:

```suggestion
| Easing | enum | `enum` that controls the `Easing`, allows you to specify a transfer function that controls how animations speed up or slow down. You can find more details on [Easing here](/dotnet/maui/user-interface/animation/easing) |
```

Reply:
Even better I didn't realise that was there and WOW the MAUI docs have moved on since I last looked.

Reply:
Yeah @davidbritch is smashing it over in the .NET MAUI docs!